doc/source/methods/spotthediffdrift.ipynb

```diff
@@ -0,0 +1,221 @@
+    "\n",
+    "* `l1_reg`: Strength of l1 regularisation to apply to the differences.\n",
+    " \n",
+    "* `binarize_preds`: Whether to test for discrepency on soft  (e.g. probs/logits) model predictions directly with a K-S test or binarise to 0-1 prediction errors and apply a binomial test.\n",
+    "\n",
+    "* `train_size`: Optional fraction (float between 0 and 1) of the dataset used to train the classifier. The drift is detected on *1 - train_size*. Cannot be used in combination with `n_folds`.\n",
+    "\n",
+    "* `n_folds`: Optional number of stratified folds used for training. The model preds are then calculated on all the out-of-fold instances. This allows to leverage all the reference and test data for drift detection at the expense of longer computation. If both `train_size` and `n_folds` are specified, `n_folds` is prioritized.\n",
+    "\n",
+    "* `retrain_from_scratch`: Whether the classifier should be retrained from scratch for each set of test data or whether it should instead continue training from where it left off on the previous set.\n",
+    "\n",
+    "* `seed`: Optional random seed for fold selection.\n",
+    "\n",
+    "* `optimizer`: Optimizer used during training of the kernel. From `torch.optim` for PyTorch and `tf.keras.optimizers` for TensorFlow.\n",
+    "\n",
+    "* `learning_rate`: Learning rate for the optimizer.\n",
+    "\n",
+    "* `batch_size`: Batch size used during training of the kernel.\n",
+    "\n",
+    "* `preprocess_batch_fn`: Optional batch preprocessing function. For example to convert a list of generic objects to a tensor which can be processed by the kernel.\n",
+    "\n",
+    "* `epochs`: Number of training epochs for the kernel.\n",
+    "\n",
+    "* `verbose`: Verbosity level during the training of the kernel. 0 is silent and 1 prints a progress bar.\n",
+    "\n",
+    "* `train_kwargs`: Optional additional kwargs for the built-in TensorFlow (`from alibi_detect.models.tensorflow import trainer`) or PyTorch (`from alibi_detect.models.pytorch import trainer`) trainer functions.\n",
+    "\n",
+    "* `dataset`: Dataset object used during training of the kernel. Defaults to `alibi_detect.utils.pytorch.data.TorchDataset` (an instance of `torch.utils.data.Dataset`) for the PyTorch backend and `alibi_detect.utils.tensorflow.data.TFDataset` (an instance of `tf.keras.utils.Sequence`) for the TensorFlow backend. For PyTorch, the dataset should only take the windows x_ref and x_test as input, so when e.g. *TorchDataset* is passed to the detector at initialisation, during training *TorchDataset(x_ref, x_test)* is used. For TensorFlow, the dataset is an instance of `tf.keras.utils.Sequence`, so when e.g. *TFDataset* is passed to the detector at initialisation, during training *TFDataset(x_ref, x_test, batch_size=batch_size, shuffle=True)* is used. x_ref and x_test can be of type np.ndarray or List[Any].\n",
```

Comment:
This is the description for LearnedKernelDrift. I think for SpotTheDiffDrift the description from ClassifierDrift should be used since we pass (x, y) to the dataset, not (x_ref, x_test).